src/legacy/core_plugins/tile_map/public/__tests__/coordinate_maps_visualization.js

```diff
@@ -111,12 +120,11 @@ describe('CoordinateMapsVisualizationTest', function () {
   describe('CoordinateMapsVisualization - basics', function () {
 
     beforeEach(async function () {
-
       setupDOM('512px', '512px');
 
       imageComparator = new ImageComparator();
       vis = new Vis(indexPattern, {
-        type: 'region_map'
+        type: 'tile_map'
```

Comment:
Thanks for fixing. This was nasty. This just used to work because it was using the vis-object only for the configuration params when calling `new CoordinateMapsVisualization(domNode, vis)`.